go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/filecoin-project/go-jsonrpc v0.6.0
 	github.com/filecoin-project/go-padreader v0.0.1
 	github.com/filecoin-project/go-paramfetch v0.0.4
-	github.com/filecoin-project/go-state-types v0.15.0-dev
+	github.com/filecoin-project/go-state-types v0.15.0-dev.0.20240930040600-f1e9746dbeaa
```

Comment:
Reminder to update to the tagged value when its out

Reply:
I will wait for a tag before I update this PR and merge it. That will depend on getting https://github.com/filecoin-project/go-state-types/pull/314 merged (FYI @rjan90).